aten/src/ATen/native/ChanelShuffle.cpp

```diff
@@ -54,7 +54,7 @@ Tensor channel_shuffle(const Tensor& self, int64_t groups) {
       .reshape(self.sizes());
   return namedinference::propagate_names_if_nonempty(
       output_tensor,
-      self.names());
+      self.has_names() ? self.names() : std::vector<Dimname>());
```

Comment:
Can you add a test somewhere to confirm this solves the problem? Maybe in `test/test_nn.py` or `test/test_named_tensor`

Reply:
Yes, I will try. Maybe it will take some time as I am not familiar with PyTorch testing

Reply:
Here's an example of how to test for the existence of no warnings:
https://github.com/pytorch/pytorch/blob/75eb026e07062e8539c3b6e96d6100ac9378a292/test/test_torch.py#L2898-L2902

Reply:
@h6197627 You can add the no-warning check to the existing `channel_shuffle` test here:
https://github.com/pytorch/pytorch/blob/ba320cec6bc0e20855f8f6ce74f4248dcfafff5e/test/test_nn.py#L9159

Reply:
@zou3519, @jbschlosser, I added generic checking for all warnings in ChannelShuffle. However, if it is not appropriate I can change it to something more specific like:
`self.assertNotIn('Named tensors and all their associated APIs are an experimental feature and subject to change.', ' '.join(map(str, w)))`